### PR TITLE
dprint format TOML files (Excluding sovereign-sdk ones)

### PR DESCRIPTION
**Pull request by rakanalh**
# Description
Trying to get this merged this time but excluding files from sovereign-sdk so that we have better and consistent TOML file format.

To see how sovereign-sdk is excluded, please look into [dprint.json](https://github.com/chainwayxyz/citrea/blob/758467ee731a227d93281cd11ccd129d587d262c/dprint.json)

### CODE REVIEW

File: .github/workflows/checks.yml [hunk 1]
```diff
@@ -92,6 +92,8 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - name: Rust Cache
         uses: Swatinem/rust-cache@v2
+      - name: Check TOML
+        uses: dprint/check@v2.2
```
Comment (kpp) [inline .github/workflows/checks.yml:96]: I am fine with dprint fmt for now, but a new check can be annoying.

**Review by kpp — APPROVED** [review]

File: .github/workflows/checks.yml (hunk 1 above)
Comment (rakanalh) [reply .github/workflows/checks.yml:96]: I have it installed & my editor would automatically execute it after i save files. You will not feel it. Or i think we can set it up pre-commit. wdyt?

Comment (eyusufatik) [reply .github/workflows/checks.yml:96]: lets go the precommit way

Comment (rakanalh) [reply .github/workflows/checks.yml:96]: Added

**Review by eyusufatik — APPROVED** [review]

**Pull request merged**